src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -168,7 +203,7 @@ public override Task SendGroupExceptAsync(string groupName, string methodName, o
             {
                 var message = CreateInvocationMessage(methodName, args);
                 var tasks = group.Values.Where(connection => !excludedIds.Contains(connection.ConnectionId))
-                    .Select(c => SafeWriteAsync(c, message));
+                    .Select(c => c.WriteAsync(message));
```

Comment:
Reuse helper method++ (and it probably take an `IEnumerable<HubConnectionContext>` to support consuming the LINQ result here.

Reply:
See comment about fixing the LINQ.